### PR TITLE
Evil fumo item + lowpoly fumo tooltip

### PR DESCRIPTION
**Pull request by Govorunb**
Please refer to the [project item](https://github.com/users/Alexejhero/projects/5/views/2?pane=issue&itemId=45512974) for design specifications

### CODE REVIEW

File: Unity/Assets/Mod/Evil Fumo/Evil Fumo Spawns.asset [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: a6ccb8c039e34e44ebdee40c7d89e4a4, type: 3}
+  m_Name: Evil Fumo Spawns
+  m_EditorClassIdentifier: 
+  spawns:
+  - game: 2
+    locations:
+    - position: {x: -327, y: 15, z: 265.5}
```
Comment (Alexejhero) [inline Unity/Assets/Mod/Evil Fumo/Evil Fumo Spawns.asset]: where does she spawn?

**Comment by Alexejhero** [comment]
Can I have a video? 

image

**Review by Alexejhero — APPROVED** [review]

File: Unity/Assets/Mod/Evil Fumo/Evil Fumo Spawns.asset (hunk 1 above)
Comment (Govorunb) [reply Unity/Assets/Mod/Evil Fumo/Evil Fumo Spawns.asset]: in a wall apparently, oops (was supposed to be right next to the other fumo)

**Comment by Govorunb** [comment]
bites if you hug for too long

https://github.com/Alexejhero/Neurosama-Subnautica-Mod/assets/3830522/a9a04f64-5ceb-425a-8e9f-d045b6fc5290

25% chance to enable alt use (poke) that does the same thing but on command

https://github.com/Alexejhero/Neurosama-Subnautica-Mod/assets/3830522/744a0d42-33d2-4535-8678-3b3f5bb6e6aa

**Comment by Alexejhero** [comment]
merge whenever ready @Govorunb

**Pull request merged**